flagsmith-jira-app/src/flagsmith.ts

```diff
@@ -166,14 +172,16 @@ export const fetchFeatures = async ({
 
 export const fetchFlags = async ({
   apiKey,
-  environmentId,
+  featureName,
+  envAPIKey,
 }: {
   apiKey: string;
-  environmentId: string | undefined;
-}): Promise<FlagModel[]> => {
+  featureName: string;
+  envAPIKey: string;
+}): Promise<FlagModel> => {
   checkApiKey(apiKey);
-  if (!environmentId) throw new ApiError("Flagsmith environment not configured", 400);
-  const path = route`/features/featurestates/?environment=${environmentId}`;
-  const data = (await flagsmithApi(apiKey, path)) as PaginatedModels<FlagModel>;
-  return await unpaginate(apiKey, data);
+  if (!featureName) throw new ApiError("Flagsmith environment not configured", 400);
```

Comment:
This error message seems wrong?

Reply:
This was already corrected in the latest version.